### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t1931

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/78faf5cd33881e6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/78faf5cd33881e6/output.txt

Total script time: 1.37 mins

Server is running on http://pr751.butyr.org/templates/basic/

File: templates/basic/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
   <link rel="stylesheet" href="../assets/plugins/zoink/popcorn.zoink.css">
   <link rel="stylesheet" href="../assets/css/popcorn.transitions.css">
   <link rel="stylesheet" href="../assets/plugins/image/popcorn.image.css">
+   <link rel="stylesheet" href="../assets/plugins/twitter/popcorn.twitter.css">
```
Comment (mjschranz) [inline templates/basic/index.html]: Fix indentation

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e04d8782a3337b9/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e04d8782a3337b9/output.txt

Total script time: 1.09 mins

Server is running on http://pr751.butyr.org/templates/basic/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/c60ac72d1f6559b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/c60ac72d1f6559b/output.txt

Total script time: 0.89 mins

Server is running on http://pr751.butyr.org/templates/basic/

**Comment by jbuck** [comment]
Staged on master: https://github.com/mozilla/butter/commit/c748d37ee24720691135873970fb4f5c7119978e

**Pull request closed without merging**